### DISCUSSION TITLE
https://unpkg.com/ 520 error code

### DISCUSSION THREAD

**Discussion by gradecost2** [Laravel-Backpack/community-forum#926] [General]
Admin panel not work normally. We have problem with cdn https://unpkg.com/
Error 520

https://prnt.sc/JjKHSWGzva_T

How fix it problem???

**Comment by tatvn** [comment]
Me too. 
Please help us!!

**Comment by pxpm** [comment]
Hey guys.

We are tracking this issue. 

Please follow https://github.com/Laravel-Backpack/community-forum/issues/928 for updates.

Cheers

**Reply by gradecost2** [reply]
Thanks! Now admin panel work!

Please advise. Can we somehow switch from cdn to using local libraries? At the same time, do not lose the opportunity to update backpack.

**Reply by pxpm** [reply]
@gradecost2 we are working on a way to cache external links and leave local files (that you usually edit while developing) out of that cache. That would have avoided the issue, unless you cleared the caches. If you did, you were not able to get the assets again since it was a CDN outage. 

It's a bit more complex than what I made it look here, since we usually do `basset:clear && basset:cache` while deploying to production, if there is a cdn outage while deploying there will be a production app down for sure. 

We need to think in the best strategy that could fix the issue at the root.
Something that separates cdn assets from local assets completely, so that we can for example, in production don't touch the cdn assets if they were not changed etc. 

We will be sure talking about this the next weeks. Subscribe to our newsletter or follow the github repo so that you get pinged when we launch something. 

Cheers
